docker/owlbot/java/Dockerfile

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
 
 # build from the root of this repo:
-FROM gcr.io/cloud-devrel-public-resources/java8
+FROM gcr.io/cloud-devrel-public-resources/java11
 
-ARG JAVA_FORMAT_VERSION=1.7
+# The formatter version used by the fmt-maven-plugin
+# https://github.com/coveooss/fmt-maven-plugin
+ARG JAVA_FORMAT_VERSION=1.11.0
```

Comment:
Probably 1.13.0. For example, https://github.com/googleapis/java-shared-config/pull/361

Reply:
Thank you. Updated.